cogeo_mosaic/backends/dynamodb.py

```diff
@@ -122,25 +167,32 @@ def write(self, overwrite: bool = False, **kwargs: Any):
                 )
             self.delete()
 
-        items = self._create_items()
-        self._write_items(items)
+        # Validation and Create a new instance
+        if not isinstance(mosaic, MosaicJSON):
+            mosaic = MosaicJSON(**dict(mosaic))
+        else:
+            mosaic = deepcopy(mosaic)
```

Comment:
because we remove `tiles` later in the method, creating a deepcopy to ensure we don't change the input mosaicJSON